Source/WebKit/UIProcess/Cocoa/WebPageWebView.swift

```diff
@@ -100,8 +100,11 @@ extension WebPageWebView {
         set { self._allowsMagnification = newValue }
     }
 
-    public func setContentOffset(_ offset: CGPoint, animated: Bool) {
```

Comment:
Would it not be convenient to still offer a version that takes an `offset: CGPoint`?

Reply:
not really, since out of the there call sites of this function, only one actually has a CGPoint, and it's trivial to convert it to its components instead of added code duplication/complexity